src/plugins/embeddable/public/lib/attribute_service/attribute_service.tsx

```diff
@@ -144,10 +144,14 @@ export class AttributeService<
       return input as ValType;
     }
     const { attributes } = await this.unwrapAttributes(input);
+    const libraryTitle = attributes.title;
     const { savedObjectId, ...originalInputToPropagate } = input;
+
     return {
       ...originalInputToPropagate,
-      attributes,
+      // by value visualizations should not have default titles and/or descriptions
+      ...{ attributes: omit(attributes, ['title', 'description']) },
+      title: libraryTitle,
```

Comment:
Now that the clone action is using `getInputAsValueType`, I moved the removal of title/description to this function since  otherwise I would have to do it in both `unlink_from_library_action` and `clone_panel_action` - by value panels should never have these attributes, so it makes more sense to have it here anyway.